src/components/Molecules/Box/Box.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Picture from '../../Atoms/Picture/Picture';
+import spacing from '../../../theme/shared/spacing';
+
+const Container = styled.div`
+  display: flex;
+  position: relative;
+  height: 100%;
+  flex-direction: column;
+  @media ${({ theme }) => theme.breakpoint('small')} {
+    flex-direction: row;
+  }
+
+  @media ${({ theme }) => theme.breakpoint('medium')} {
+    flex-direction: column;
+  }
+`;
+
+const Image = styled.div`
```

Comment:
Can you rename as it's not img tag

Reply:
Hi, it doesn't need to describe what the tag is. Same for Copy. Let's keep names simple.. only use ImageWrapper if you already used Image.